subprojects/packagefiles/boost/libs/locale/meson.build

```diff
@@ -64,6 +64,10 @@ endif
 if not cpp.check_header('iconv.h')
     locale_deps += dependency('iconv')
 endif
+iconv_dep = cpp.find_library('iconv', required: false)
```

Comment:
I don't understand why this is needed. iconv should be already available to the compiler, or the previous lines should trigger the subproject, right?. The logic used in the boost subproject should not be significantly different from the main project: https://github.com/TypesettingTools/Aegisub/blob/c6f4ea45aaf83f27b5d862acb2beeeb9c16f6cd5/meson.build#L80-L83

Reply:
Shouldn't be needed for a static build. I did a shared library build first, and boost_locale didn't link against libiconv without that.

Reply:
See https://github.com/TypesettingTools/Aegisub/pull/106